src/app.rs

```diff
@@ -2566,21 +2584,10 @@ impl TaskwarriorTui {
                                         }
                                         if t.due().is_some() {
                                             let date = t.due().unwrap();
-                                            let now = Local::now();
-                                            let date = TimeZone::from_utc_datetime(now.offset(), date);
-                                            let (sym, h, m) = get_offset_hour_minute();
                                             s = format!(
-                                                "{}due:'{:04}-{:02}-{:02}T{:02}:{:02}:{:02}{}{:02}:{:02}' ",
+                                                "{}due:{} ",
                                                 s,
-                                                date.year(),
-                                                date.month(),
-                                                date.day(),
-                                                date.hour(),
-                                                date.minute(),
-                                                date.second(),
-                                                sym,
-                                                h,
-                                                m
+                                                get_formatted_datetime(date);
```

Comment:
@kdheepak build fails here, shouldn't it be a `,`?

Reply:
Resolved in [493e320](https://github.com/kdheepak/taskwarrior-tui/pull/435/commits/493e320a35fc9b56cdedee26ef58e00a639b6043).